src/v2/providers/alerts/alerts.ts

```diff
@@ -0,0 +1,127 @@
+import { ManifestEndpoint } from '../../../common/manifest';
+import { CloudEvent, CloudFunction } from '../../core';
+import * as options from '../../options';
+
+/** Data */
+export interface FirebaseAlertData<T = any> {
+  createTime: string;
+  endTime: string;
+  payload: T;
+}
+
+/** @internal */
+export const eventType = 'firebase.firebasealerts.alerts.v1.published';
+
+/** The underlying alert type of the Firebase Alerts provider */
+export type AlertType =
+  | 'crashlytics.newFatalIssue'
+  | 'crashlytics.newNonfatalIssue'
+  | 'crashlytics.regression'
+  | 'crashlytics.stabilityDigest'
+  | 'crashlytics.velocity'
+  | 'crashlytics.newAnrIssue'
+  | 'billing.planUpdate'
+  | 'billing.automatedPlanUpdate'
+  | 'appDistribution.newTesterIosDevice'
+  | string;
+
+/** Options */
```

Comment:
nit* I found some of these comments kinda unhelpful - e.g. I can tell from the interface name that this is an option.  Can we expand on the documentations or get rid of them?

Reply:
I had those comments to break up the file into sections, but since we don't really do that anywhere else, I've made them a bit more descriptive